app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardItemViewHolder.kt

```diff
@@ -26,6 +26,9 @@ class CreditCardItemViewHolder(
     fun bind(creditCard: CreditCard) {
         credit_card_logo.setImageResource(creditCard.cardType.creditCardIssuerNetwork().icon)
 
+        credit_card_network.text = creditCard.cardType.creditCardIssuerNetwork().name
```

Comment:
Ideally, `name` would be the correct display name, but it really isn't in this case. We can do this as a followup. I will try to get a list of display names for the network names in the meantime, and we will have to add a mapping similar to https://searchfox.org/mozilla-mobile/source/android-components/components/support/utils/src/main/java/mozilla/components/support/utils/CreditCardUtils.kt#103 in Fenix to land this tomorrow.

Reply:
I never gotta a formal list of names from UX or Product. I think we should do a best effort and note that it can be localized https://github.com/mozilla-mobile/fenix/issues/19684#issuecomment-849862981.